### PR TITLE
Add header to comment page

### PR DESCRIPTION
**Pull request by shashankanandkannan**
Also made some edits to the header components to make it make more sense.

### CODE REVIEW

File: mobile/src/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "axios": "^0.20.0",
     "expo": "^38.0.10",
     "expo-constants": "~9.1.1",
-    "expo-notifications": "~0.3.3",
+    "expo-notifications": "^0.3.3",
```
Comment (shashankanandkannan) [inline mobile/src/package.json:21]: The app didn't run locally until I ran `npm install expo-notifications`. Is it okay the changes from that in this PR?

Comment (deandrebaker) [reply mobile/src/package.json:21]: @shashankanandkannan why didn't you do `npm install` instead? Did that not work? You can also discard the changes made to package.json and package-lock.json if they aren't necessary

Comment (shashankanandkannan) [reply mobile/src/package.json:21]: I did, but it didn't wor for some reason. I got an error saying it couldn't find expo-notifications, so I specifically installed it.

Comment (deandrebaker) [reply mobile/src/package.json:25]: oh wait, I think you need to do ```expo install``` instead

Comment (trentstauff) [reply mobile/src/package.json:25]: I had the same error and did `npm i` and things worked fine for me

Comment (trentstauff) [reply mobile/src/package.json:25]: not sure why it didnt fix your issue

**Review by deandrebaker — APPROVED** [review]

**Pull request merged**